test/powershell/Modules/Microsoft.PowerShell.Core/Enter-PSHostProcess.Tests.ps1

```diff
@@ -32,16 +26,6 @@ Describe "Enter-PSHostProcess tests" -Tag Feature {
         }
     }
 
-    # Skip on Appveyor due to PSReadline issue.
-    It "Can enter and exit another PSHost" -Skip:$IsAppVeyor {
```

Comment:
Why the test was removed? We need to remove only `-Skip:$IsAppVeyor`
The same for next test

Reply:
Because this test is only for appveyor as said in the comment (sorry if I'm wrong)

Reply:
The comment says that we _SKIP_ the test on Appveyor.

Reply:
Ohhh okay, I'll add it back without the appveyor check.